docs-src/docs/guides/terraform_beyond_layer0.md

```diff
@@ -0,0 +1,329 @@
+# Deployment guide: Terraform beyond Layer0
+In this example, you will learn how you can use Terraform to create a Layer0 service as well as supporting infrastructure in the form of a persistent data store. This example does not cover the details of the application being deployed but focuses on how you can combine Layer0 and other Terraform providers as part of a Terraform workflow.
```

Comment:
[suggestion] In this example, we'll learn how you can use Terraform to create a Layer0 service as well as a [persistent data store](https://aws.amazon.com/dynamodb/).